src/observers/telemetryObserver.ts

```diff
@@ -34,8 +37,8 @@ export class TelemetryObserver {
             case EventType.BuildCompleted:
                 this.handleBuildCompleted(<BuildCompleted>event);
                 break;
-            case EventType.BuildCacheSizeCalculated:
-                this.handleBuildCacheSize(<BuildCacheSizeCalculated>event);
+            case EventType.CalculateBuildCacheSize:
```

Comment:
Can we reuse the `DocfxRestoreCompleted`?

Reply:
I think we should use `DocfxRestoreStarted`